src/role-access/containers/specific-access/specific-access-approved/specific-access-approved.component.ts

```diff
@@ -7,7 +7,8 @@ import * as fromFeature from '../../../store';
 
 @Component({
   selector: 'exui-specific-access-approved',
-  templateUrl: './specific-access-approved.component.html'
+  templateUrl: './specific-access-approved.component.html',
+  styleUrls: ['./specific-access-approved.component.scss']
```

Comment:
This should not be needed if you eliminate the padding issue

Reply:
have removed